radio/src/pulses/multi_arm.cpp

```diff
@@ -46,7 +46,7 @@ static void sendByteMulti(uint8_t b)
     parity = parity ^ (b & 1);
     b >>= 1;
   }
-  putDsm2SerialBit(parity);      // Even Parity bit
+  putDsm2SerialBit(!parity);     // Even Parity bit
```

Comment:
It's only for PPM_PIN_SERIAL, which I believe isn't used on any other platforms, which explains it was never noticed that it was wrong.